connector_helpscout/models/helpscout_tag/importer.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+from odoo.addons.connector.components.mapper import mapping, only_create
+
+
+tag_color_map = {
+    (211, 211, 211): 0,
+    (0, 128, 0): 1,
+    (255, 255, 0): 2,
+    (255, 0, 0): 4,
+    (128, 0, 128): 5,
+    (0, 0, 255): 6,
+}
+
+
+class HelpScoutTagImportMapper(Component):
+    _name = 'helpscout.import.mapper.tag'
+    _inherit = 'helpscout.import.mapper'
+    _apply_on = 'helpscout.tag'
+
+    direct = [('tag', 'name'),
+              ('slug', 'helpscout_slug'),
+              ('created_at', 'backend_date_created'),
```

Comment:
This is already done in the default mapper https://github.com/LasLabs/odoo-connector-helpscout/blob/10.0/connector_helpscout/components/mapper.py